Integrations/EWSv2/EWSv2.yml

```diff
@@ -60,8 +60,8 @@ configuration:
   name: defaultServerVersion
   required: false
   type: 0
-- defaultvalue: 'true'
-  display: Trust any certificate (unsecure)
+- defaultvalue: ''
```

Comment:
Changing the default value to false can cause ssl verification error at users that already configured `EWS v2`, so it should be taken into consideration.

Reply:
After an instance's parameter value was set (with default value or not), changing the parameter's default value in the .yml file won't affect the current value.